testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -1084,9 +1084,8 @@ fn bitcoind_resubmission_test() {
     .unwrap();
 
     let burn_tip = burnchain_db.get_canonical_chain_tip().unwrap();
-    let last_burn_block = burnchain_db
-        .get_burnchain_block(&burn_tip.block_hash)
-        .unwrap();
+    let last_burn_block =
+        BurnchainDB::get_burnchain_block(burnchain_db.conn(), &burn_tip.block_hash).unwrap();
```

Comment:
Why change these methods from instance methods to class methods? Unless there's a pressing reason to do so, the class methods seem worse, because they require more boiler plate _and_ they eliminate useful typing information (replacing the typed database connection with opaque &Connection references).

Reply:
The `get_burnchain_block()` method is needed in both a connection context and transaction context (in `src/burnchains/affirmation.rs`).  This was meant to *reduce* boiler-plate.